discordbio/dist/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "author": {
+    "name": "MauriceX",
+    "id": "268768664411439115"
+  },
+  "service": "Discord.bio",
+  "description": {
+    "en": "Create a profile in under 30 seconds.",
+    "de": "Erstelle ein Profil in weniger als 30 sekunden."
+  },
+  "url": ["discord.bio"],
+  "version": "1.0",
+  "logo": "https://i.imgur.com/19jXt3Z.jpg",
```

Comment:
minimal comment (not necessary to change) but i think the logo is a bit giant xd

Reply:
Changed. :D